### PR TITLE
updated PR template regarding issue #290

### PR DESCRIPTION
**Pull request by AshishSharma1203**
# Zero To Mastery - Webblocks 2022 📦📦

## PR contains:

- I have updated the PR template regarding issue #290 

## Breaking changes
None 
## Screenshots
None

### CODE REVIEW

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -1,14 +1,56 @@
...
+- Chnage 2
+
+## Additional Info
+- Any additional information or context
+ -->
+## Checklist
+
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] ✅ My code follows the code style of this project.
```
Comment (sophiabrandt) [inline docs/pull_request_template.md:28]: While I like these checkboxes, there are no style guides in this repo. So this checkbox makes no sense.

File: docs/pull_request_template.md [hunk 2]
```diff
@@ -1,14 +1,56 @@
...
+- [ ] 📝 My change requires a change to the documentation.
+- [ ] 🎀 I have updated the documentation accordingly.
+- [ ] 🌟 ed the repo
```
Comment (sophiabrandt) [inline docs/pull_request_template.md:31]: Contributors shouldn't be forced to star the repo.

File: docs/pull_request_template.md (hunk 1 above)
Comment (AshishSharma1203) [reply docs/pull_request_template.md:29]: I have added a line to instruct on how to add a check mark on this.

File: docs/pull_request_template.md (hunk 2 above)
Comment (AshishSharma1203) [reply docs/pull_request_template.md]: I have removed this checklist option .

**Comment by AshishSharma1203** [comment]
I have made the requested change. Sorry for the late changes made, I had my exam .

File: docs/pull_request_template.md (hunk 1 above)
Comment (sophiabrandt) [reply docs/pull_request_template.md:29]: Yes, but without a style guide we cannot ask people to check that they followed the style guide.
I suggest removing this line.

File: docs/pull_request_template.md [hunk 3]
```diff
@@ -1,14 +1,57 @@
...
-- IF YOU HAVE NO SCREENSHOTS, ENTER 'None'
\ No newline at end of file
+- IF YOU HAVE NO SCREENSHOTS, ENTER 'None'
+
+## Related Issue
+- Mention issue for which this pull request is a part of (please mention issue number # )
+
+## Proposed Changes
+- Change 1
+- Chnage 2
```
Comment (sophiabrandt) [inline docs/pull_request_template.md:19]: Typo in `change`

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

File: docs/pull_request_template.md (hunk 1 above)
Comment (AshishSharma1203) [reply docs/pull_request_template.md]: removed that from the list.

**Pull request merged**